apps/web/app/composables/useResetProductPageModal/useResetProductPageModal.ts

```diff
@@ -1,20 +1,34 @@
 import type {
   UseResetProductPageModalReturn,
   UseResetProductPageModalState,
+  ResetType,
 } from '~/composables/useResetProductPageModal/types';
 
 export const useResetProductPageModal: UseResetProductPageModalReturn = () => {
   const state = useState<UseResetProductPageModalState>(`resetProductPageModal`, () => ({
     unlinkModalOpen: false,
-    resetType: null,
+    resetType: '' as ResetType,
+    loading: false,
   }));
 
-  const toggleResetModal = (value: boolean, resetType?: string) => {
-    if (resetType) {
-      state.value.resetType = resetType;
-    }
+  const toggleResetModal = (value: boolean, resetType: ResetType = '') => {
+    state.value.resetType = resetType;
     state.value.unlinkModalOpen = value;
   };
+  const resetBlocks = async (identifier: number, type: ResetType) => {
+    state.value.loading = true;
+    const { send } = useNotification();
+
+    try {
+      await useSdk().plentysystems.deleteBlocks({ identifier, type });
+      send({ type: 'positive', message: 'Blocks reseted successfully. Please refresh the page.' });
```

Comment:
```suggestion
      send({ type: 'positive', message: 'Blocks reset successfully. Please refresh the page.' });
```
The past tense of "reset" is "reset".

Also, we should avoid hard coding the translation strings. Let's move them to `utils/editorTranslations.en.json` and `editorTranslations.de.json` for now (English string both times).